yabt/buildcontext.py

```diff
@@ -396,8 +399,16 @@ def run_in_buildenv(
         docker_run.extend(cmd)
         logger.info('Running command in build env "{}" using command {}',
                     buildenv_target_name, docker_run)
-        # TODO(bergden): accumulate error prints
-        return run(docker_run, check=True, **kwargs)
+        if 'stderr' not in kwargs:
+            kwargs['stderr'] = PIPE
+        if 'stdout' not in kwargs:
+            kwargs['stdout'] = PIPE
+        result = run(docker_run, check=True, **kwargs)
```

Comment:
it can be risky set stdout/stderr to PIPE (see NOTE [here](https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate)).

I guess it's not going to be an issue for us, but need to pay attention to memory behavior and consider changing this to temp files instead of pipes if it becomes an issue.

consider at least adding a comment with TODO here to keep it in mind

Reply:
Added the TODO.